VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -59,7 +78,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
             public boolean onQueryTextSubmit(String query) {
                 String baseUrl = environment.getConfig().getCourseDiscoveryConfig().getCourseSearchUrl();
                 String searchUrl = buildQuery(baseUrl, query, logger);
-                searchView.clearFocus();
+                searchView.onActionViewCollapsed();
```

Comment:
No, this is for collapsing the `SearchView`. I couldn't find any method to do it directly before, so I suggested this hack.